common/rfb/ZRLEEncoder.cxx

```diff
@@ -31,7 +31,7 @@
 
 using namespace rfb;
 
-IntParameter zlibLevel("ZlibLevel","Zlib compression level",-1);
+IntParameter zlibLevel("ZlibLevel","[DEPRECATED] Zlib compression level",-1);
```

Comment:
We should also stop using it so we get a clear switch to using the normal compression setting.